collector/wifi_linux.go

```diff
@@ -65,14 +65,14 @@ func NewWifiCollector() (Collector, error) {
 	)
 
 	var (
-		labels = []string{"device"}
+		labels = []string{"device", "hw"}
```

Comment:
I think `hw` is nice and short, but not very descriptive. We generally recommend more descriptive names. How about `station_address`.

Reply:
From nl80211.h, it is called
 ```* @NL80211_ATTR_MAC: MAC address (various uses)```

On linux  ```iw  dev wlan1 station dump``` command output, it is not explicity named.

I think ```station_address``` might be a bit repetitive given the statistics are named ```node_station_``` but any of the following would be fine:

```mac```
```hardware_address```
```hw_address```
or even just ```address```

Would one of be acceptable?

Reply:
True, how about `mac_address`, to match the datasource.